NitroxModel/DataStructures/GameLogic/AbsoluteEntityCell.cs

```diff
@@ -88,30 +127,14 @@ public Int3 GetCellSize(Int3 blocksPerBatch)
             return GetCellSize(Level, blocksPerBatch);
         }
 
-        public static Int3 GetCellSize(int level, Int3 blocksPerBatch)
-        {
-            // Our own implementation for BatchCells.GetCellSize, that works on the server and client.
-            return blocksPerBatch / GetCellsPerBlock(level);
-        }
-
         public int GetCellsPerBlock()
         {
             return GetCellsPerBlock(Level);
         }
 
-        public static int GetCellsPerBlock(int level)
+        protected bool Equals(AbsoluteEntityCell other)
         {
-            switch (level)
-            {
-                case 0:
-                    return 10;
-                case 1:
-                case 2:
-                case 3:
-                    return 5;
-                default:
-                    throw new Exception();
-            }
+            return Equals(BatchId, other.BatchId) && Equals(CellId, other.CellId) && Level == other.Level;
```

Comment:
Can other be null here?  For example, if this code path is invoked from == or != operators?

Reply:
Technically can be but it's protected sn only called by the overridden public equals against type `object`.